test/scripts/openshift-ci/run-e2e-tests.sh

```diff
@@ -77,6 +77,9 @@ kustomize build $PROJECT_ROOT/config/overlays/test | \
   oc apply -f -
 oc wait --for=condition=ready pod -l control-plane=kserve-controller-manager -n kserve --timeout=300s
 
+echo "Installing odh-model-controller"
+kustomize build $PROJECT_ROOT/test/scripts/openshift-ci | oc apply -f -
```

Comment:
This can be simplified `oc apply -k $PROJECT_ROOT/test/scripts/openshift-ci`

Reply:
Fixed.